app/Http/Controllers/ContactController.php

```diff
@@ -35,8 +36,18 @@ public function send(Request $request)
             'email' => 'required|email',
             'subject' => 'required',
             'message' => 'required',
+            'recaptcha' => 'required',
         ]);
 
+        $recaptcha = new ReCaptcha(config('services.google.recaptcha_secret'));
+        $response = $recaptcha->verify($request->recaptcha, $request->ip());
+
+        if (!$response->isSuccess() || $response->getScore() < 0.6) {
+            return redirect()->back()
+                ->with(['success' => false])
+                ->withErrors(['recaptcha' => ['Invalid response to spam filters.']]);
```

Comment:
We should probably add with input here as well so it repopulates the form.
```suggestion
                ->withErrors(['recaptcha' => ['Invalid response to spam filters.']])
                ->withInput();
```